src/test/java/core/basesyntax/service/TransactionInvocationImplTest.java

```diff
@@ -0,0 +1,109 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.operations.BalanceTransactionExecutor;
+import core.basesyntax.operations.PurchasingTransactionExecutor;
+import core.basesyntax.operations.ReturningTransactionExecutor;
+import core.basesyntax.operations.SupplingTransactionExecutor;
+import core.basesyntax.operations.TransactionExecutor;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class TransactionInvocationImplTest {
+    @Rule
+    public ExpectedException thrownRule = ExpectedException.none();
+
+    @After
+    public void clearStorage() {
+        Storage.FRUIT_MAP.clear();
+    }
+
+    @Test
+    public void involveBalanceTransaction_ok() {
```

Comment:
```suggestion
    public void invokeTransaction_balance_ok() {
```
same for other names